src/main/java/org/maxgamer/rs/core/Core.java

```diff
@@ -272,7 +261,6 @@ private static void shutdown() {
             getThreadPool().awaitTermination(5, TimeUnit.SECONDS);
 		}
 		catch (InterruptedException e) {}
-		Log.close();
 	}
 	
 	public synchronized static Timings getTimings() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)